omnigibson/utils/usd_utils.py

```diff
@@ -637,6 +637,17 @@ def add_asset_to_stage(asset_path, prim_path):
     # Make sure prim was loaded correctly
     assert prim, f"Failed to load {asset_type.upper()} object from path: {asset_path}"
 
+    # Apply any forced light intensity updates.
+    if gm.FORCE_USD_LIGHT_INTENSITY:
```

Comment:
I don't think this should be done here in `add_asset_to_stage` -- imo it should be handled at the object level, e.g.: `entity_prim.py` or `base_object.py`. What do you think?